CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
+
+* @seanyoung @xermicus @LucasSte
```

Comment:
Feel free to add yourself

Reply:
Thanks. I added myself but this causes an error on github because I don't have write access to the repository.